tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -31,96 +31,120 @@
-        self.nodes[ROUTER1].add_whitelist(self.nodes[LEADER].get_addr64())
-        self.nodes[ROUTER1].add_whitelist(self.nodes[ED1].get_addr64())
+        self.nodes[ROUTER1].add_whitelist(self.nodes[DUT_LEADER].get_addr64())
+        self.nodes[ROUTER1].add_whitelist(self.nodes[MED1].get_addr64())
         self.nodes[ROUTER1].enable_whitelist()
         self.nodes[ROUTER1].set_router_selection_jitter(1)
 
         self.nodes[ROUTER2].set_panid(0xface)
         self.nodes[ROUTER2].set_mode('rsdn')
-        self.nodes[ROUTER2].add_whitelist(self.nodes[LEADER].get_addr64())
-        self.nodes[ROUTER2].add_whitelist(self.nodes[ED2].get_addr64())
+        self.nodes[ROUTER2].add_whitelist(self.nodes[DUT_LEADER].get_addr64())
+        self.nodes[ROUTER2].add_whitelist(self.nodes[SED1].get_addr64())
         self.nodes[ROUTER2].enable_whitelist()
         self.nodes[ROUTER2].set_router_selection_jitter(1)
 
-        self.nodes[ED1].set_panid(0xface)
-        self.nodes[ED1].set_mode('rsn')
-        self.nodes[ED1].add_whitelist(self.nodes[ROUTER1].get_addr64())
-        self.nodes[ED1].enable_whitelist()
+        self.nodes[MED1].set_panid(0xface)
+        self.nodes[MED1].set_mode('rsn')
+        self.nodes[MED1].add_whitelist(self.nodes[ROUTER1].get_addr64())
+        self.nodes[MED1].enable_whitelist()
 
-        self.nodes[ED2].set_panid(0xface)
-        self.nodes[ED2].set_mode('rsn')
-        self.nodes[ED2].add_whitelist(self.nodes[ROUTER2].get_addr64())
-        self.nodes[ED2].enable_whitelist()
+        self.nodes[SED1].set_panid(0xface)
+        self.nodes[SED1].set_mode('rsn')
+        self.nodes[SED1].add_whitelist(self.nodes[ROUTER2].get_addr64())
+        self.nodes[SED1].enable_whitelist()
 
-        self.nodes[ED3].set_panid(0xface)
-        self.nodes[ED3].set_mode('rsn')
-        self.nodes[ED3].add_whitelist(self.nodes[LEADER].get_addr64())
-        self.nodes[ED3].enable_whitelist()
+        self.nodes[MED3].set_panid(0xface)
+        self.nodes[MED3].set_mode('rsn')
+        self.nodes[MED3].add_whitelist(self.nodes[DUT_LEADER].get_addr64())
+        self.nodes[MED3].enable_whitelist()
+
+        self.sniffer = config.create_default_thread_sniffer()
+        self.sniffer.start()
 
     def tearDown(self):
+        self.sniffer.stop()
+        del self.sniffer
+
         for node in list(self.nodes.values()):
             node.stop()
         del self.nodes
 
     def test(self):
-        self.nodes[LEADER].start()
-        self.nodes[LEADER].set_state('leader')
-        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+        # 1
+        self.nodes[DUT_LEADER].start()
+        self.nodes[DUT_LEADER].set_state('leader')
+        self.assertEqual(self.nodes[DUT_LEADER].get_state(), 'leader')
 
-        self.nodes[ROUTER1].start()
-        time.sleep(5)
-        self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
+        for DEV in range(ROUTER1, MED3 + 1):
+            self.nodes[DEV].start()
```

Comment:
Follow the other iterator naming in other scripts and using `i` instead of `DEV`?